modules/bclconvert/main.nf

```diff
@@ -0,0 +1,84 @@
+process BCLCONVERT {
+    tag 'demultiplexing'
+    label 'process_high'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bcl-convert. Please use docker or singularity containers."
+    }
+    container "nfcore/bclconvert:3.9.3"
+
+    input:
+    path samplesheet
+    path run_dir
+
+    output:
+    path "*.fastq.gz"               ,emit: fastq
+    path "Reports/*.{csv,xml,bin}"  ,emit: reports
+    path "Logs/*.{log,txt}"         ,emit: logs
+    path "InterOp/*.bin"            ,emit: interop
+    path "versions.yml"             ,emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    bcl-convert \
+        $args \\
+        --output-directory $PWD \\
```

Comment:
I think this need to be escaped

Reply:
Nice catch! Fixed it

Reply:
Any reason not to have a possible custom prefix here as well?

Reply:
The `$PWD` only designates an output directory but has no effect on the names of the data whatsoever.
Personally I don't really see the benefit of introducing another subdir into the mix, especially considering `bcl-convert` can also make its own subdirs, depending on the settings.

It's possible, but I wouldn't do it. We'd also have to modify the customisations after the process to collect the `InterOp` bin files

Reply:
However, if there is a consensus or guideline as to how this should be handled, I think that should be followed.

Reply:
Have you tried `.` instead of `$PWD`? That's more common to use, I agree on not having a custom prefix.

Reply:
I thought of using `.` but personally I think `PWD` is more readable. A single dot is very easily looked over (in my experience), hence my perference for `$PWD`

Reply:
I'm just trying to keep a consistent style, `.` is more common in the `nf-core/modules`. 

After looking at this again, should the `$` be escaped with `\`?